lib/evm/precompiles/04-identity.ts

```diff
@@ -8,9 +8,9 @@ export default function(opts: PrecompileInput): ExecResult {
 
   const data = opts.data
 
-  const gasUsed = new BN(opts._common.param('gasPrices', 'identity'))
-  gasUsed.iadd(
-    new BN(opts._common.param('gasPrices', 'identityWord')).imuln(Math.ceil(data.length / 32)),
+  let gasUsed = new BN(opts._common.param('gasPrices', 'identity'))
```

Comment:
The precompiles don't affect stack elements, so probably no need to change them?

Reply:
Yeah, I noticed the same. I changed them with the idea of also proposing a linting rule forbidding them. Without that proposal nor an explanation they must look confusing, sorry about that.